atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -301,6 +310,8 @@ def render_request(self):
         request = None
         if self.packages:
             request = self.render_packages_request()
+            if self.arches:
+                request.update({'arches': self.arches})
```

Comment:
maybe just : request['arches'] = self.arches 
?

Reply:
arches is an optional keyword. I'd like to leave it that way.
the handling for arches will change somewhat in osbs-5259 anyway.

Reply:
what I've meant, is to keep the 'if self.arches' but instead of update, just set that key directly, as update will do that anyway, but since we are updating only 1 key, we can just as well set it directly

Reply:
Oh, okay.

Fixed.